### ISSUE TITLE
`npm i @angular/core -DE` results in @angular/core@4.4.0-RC.0 installed

### ISSUE THREAD

**Issue by plantain-00** [angular/angular#19013]
## I'm submitting a...

<pre><code>
[x] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

`npm view @angular/core dist-tags`

```
{ latest: '4.4.0-RC.0', next: '5.0.0-beta.6' }
```

## Expected behavior

```
{ latest: '4.3.6', next: '5.0.0-beta.6' }
```

## Minimal reproduction of the problem with instructions

`npm i @angular/core -DE` or `npm view @angular/core dist-tags`

## What is the motivation / use case for changing the behavior?

Install angular, find a rc installed rather than the latest stable one

## Environment

<pre><code>
Angular version: 4.3.6

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by dfederm** [comment]
+1, it seems weird for a prerelease package to be @latest.

Based on the rules for semver, packages which specify ^4.0.0 won't match 4.4.0-RC.0

This leads to a whole bunch of npm warnings and has also caused some issues when packages (improperly) specify angular ^4.0.0 as a dependency instead of a peerDependency, which looks to be somewhat common, which causes those packages to pull in their own version of angular and thus break at runtime with errors along the lines of modules not being modules (different @NgModule versions) or services having no provide (different @Injectable versions).

Example of npm warnings:
```
npm WARN @markpieszak/ng-application-insights@2.3.2 requires a peer of @angular/router@^4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @ng-bootstrap/ng-bootstrap@1.0.0-beta.4 requires a peer of @angular/core@^4.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN @ng-bootstrap/ng-bootstrap@1.0.0-beta.4 requires a peer of @angular/common@^4.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN @ng-bootstrap/ng-bootstrap@1.0.0-beta.4 requires a peer of @angular/forms@^4.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN codelyzer@3.1.2 requires a peer of @angular/compiler@^2.3.1 || >=4.0.0-beta <5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN codelyzer@3.1.2 requires a peer of @angular/core@^2.3.1 || >=4.0.0-beta <5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-adsense@4.0.12 requires a peer of @angular/core@>=4.3.0 <=6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-adsense@4.0.12 requires a peer of @angular/common@>=4.3.0 <=6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-charts@1.6.0 requires a peer of @angular/common@^2.3.0 || >=4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-charts@1.6.0 requires a peer of @angular/core@^2.3.0 || >=4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-clipboard@8.0.4 requires a peer of @angular/core@>=4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-clipboard@8.0.4 requires a peer of @angular/common@>=4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-clipboard@8.0.4 requires a peer of @angular/platform-browser@>=4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-window-token@0.0.2 requires a peer of @angular/core@>=2.4.1 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-window-token@0.0.2 requires a peer of @angular/common@>=2.4.1 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-window-token@0.0.2 requires a peer of @angular/platform-browser@>=2.4.1 but none is installed. You must install peer dependencies yourself.
```

**Comment by erika9star** [comment]
v4.4.0-RC.0 is listed as the latest stable version on https://angular.io/guide/quickstart
 
![image](https://user-images.githubusercontent.com/22220160/30292161-2c963f8e-9703-11e7-9bf2-9d86616c4766.png)

**Comment by dfederm** [comment]
@erika9star Yes, that is exactly what we're pointing out as the bug. A prerelease package should not be marked as the latest (stable). As noted above, it's causing all sorts of issues when it is.

**Comment by sarunint** [comment]
@robwormald said to me on Gitter yesterday that this is wrong and would be fixed.

**Comment by plantain-00** [comment]
Seems fixed, it's `{ latest: '4.4.1', next: '5.0.0-beta.7' }` now
